### PR TITLE
refactor nginx proxy service

### PR DESCRIPTION
**Pull request by naumvd95**
- increase code visibility
- provide possibility of managing certs quickly and easily
- remove hardcode
- provide valid instructions as comments and also as release notes(see ex below)
```
Configure variables in .env file from Nginx section:
NGINX_VHOSTNAME - domain name for Kqueen service. Should be equal with domain name in generated certificates.
NGINX_SSL_CERTIFICATE_DIR - Mapped directory for certificates forwarding into docker container.

Check proxy service configuration in docker-compose.production.yml. Pay attention on following variables:

VHOSTNAME - domain name for Kqueen service. Should be equal with domain name in generated certificates. Default: Using variable from .env file, named NGINX_VHOSTNAME
SSL_CERTIFICATE_DIR - Mapped directory for certificates forwarding into docker container. Default: Using variable from .env file, named NGINX_SSL_CERTIFICATE_DIR/NGINX_VHOSTNAME
SSL_CERTIFICATE_PATH - Path for cert+key certificate. Default: $SSL_CERTIFICATE_DIR/fullchain.cer
SSL_CERTIFICATE_KEY_PATH -  Path for certificate key. Default: $SSL_CERTIFICATE_DIR/$VHOSTNAME.key
SSL_TRUSTED_CERTIFICATE_PATH - Path for certificate only. Default: $SSL_CERTIFICATE_DIR/ca.cer

Check that local certificates naming equal to defined in variables.

Map volumes with certificates. Pay attention that destination path should be equal with SSL_CERTIFICATE_DIR. Example:

    volumes:
      - /your/local/cert/storage/kqueen/certs/:${NGINX_SSL_CERTIFICATE_DIR}/${NGINX_VHOSTNAME}:ro

Build proxy service image:

docker-compose -f docker-compose.production.yml build --no-cache

Re-run production services:

docker-compose -f docker-compose.yml -f docker-compose.production.yml  up --force-recreate
```
duplicated from https://docs.google.com/document/d/1Bnor6D78fVzOndie52wFHVVbgV0daTOHrVP1IgKZvic/edit?usp=sharing

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Domain name for service. Should be equal with name in generated ssl-certificate
+NGINX_DEBUG=True
+# Domain name for service. Should be equal with name in generated ssl-certificate
+NGINX_VHOSTNAME=demo.kqueen.net
+# Directory path for certificates in container.Finally it look like $NGINX_SSL_CERTIFICATE_DIR/$NGINX_VHOSTNAME
+NGINX_SSL_CERTIFICATE_DIR=/mnt/letsencrypt
```
Comment (katyafervent) [inline .env]: Should name be more specific? what if some other images will need env variables?

Comment (naumvd95) [reply .env]: .env file its common default file for running compose-deloyments

**Comment by naumvd95** [comment]
@atengler when we should review and merge it?

**Comment by naumvd95** [comment]
@tomkukral 
^^

**Pull request merged**